README.md

```diff
@@ -62,7 +62,7 @@ Run the service code locally (serving at http://localhost:3000 by default):
 ```bash
 pip install torch transformers  # additional dependencies for local run
 
-bentoml serve service.py:Summarization
+bentoml serve
```

Comment:
Maybe mention that `service.py` is a convention, otherwise you can specify it like `bentoml serve <module_name>:<attr_name>`

Reply:
Updated. I think we can just mention that in the Services doc directly